coldfusion/README.md

```diff
@@ -20,9 +20,9 @@ Along with the conventions reported below, we follow the [Ortus ColdFusion (CFML
 **Right:**
 ```html
 <cfif a EQ 1>
-    <cfset classNames="component">
+	<cfset classNames="component">
 <cfelse>
-    <cfset classNames="component component--modifier">
+	<cfset classNames="component component--modifier">
```

Comment:
This example should also comply with the 'close self-closing tags' convention below